tools/compstatic/Makefile

```diff
@@ -0,0 +1,16 @@
+CC := gcc
+CFLAGS := -O3
+
+SRCS = $(wildcard *.c)
+INCLUDES = $(wildcard *.h)
+
+.PHONY: all clean
+
+all: compstatic
+	@:
+
+compstatic: $(SRCS) $(INCLUDES)
+	$(CC) $(CFLAGS) -o $@ $^
```

Comment:
`$^` here expands to `$(SRCS) $(INCLUDES)`, which causes issues when compiling with clang (gcc alias on mac).

Reply:
Caveat: editing a header file won't trigger a rebuild.